reputation/reputation_contract.go

```diff
@@ -23,20 +23,15 @@ func _deploy(data interface{}, isUpdate bool) {
 	if isUpdate {
 		args := data.([]interface{})
 		common.CheckVersion(args[len(args)-1].(int))
+		storage.Delete(ctx, notaryDisabledKey)
 		return
 	}
 
-	args := data.(struct {
+	_ = data.(struct {
+		//TODO(@acid-ant): remove notaryDisabled in future version
 		notaryDisabled bool
 	})
```

Comment:
I would check that `notaryDisabled` is false and panic otherwise in all `_deploy` functions.

Reply:
Sounds good, updated.